native/packages/react-native-bpk-component-button/src/__snapshots__/BpkButton-test.ios.js.snap

```diff
@@ -1534,128 +1534,6 @@ exports[`iOS BpkButton should support the "large" property 1`] = `
 </BVLinearGradient>
 `;
 
-exports[`iOS BpkButton should support the "selected" property 1`] = `
-<BVLinearGradient
-  colors={
-    Array [
-      4278230461,
-      4278226088,
-    ]
-  }
-  end={undefined}
-  locations={null}
-  start={undefined}
-  style={
-    Array [
-      Array [
-        Object {
-          "borderRadius": 40,
-          "height": 32,
-        },
-        undefined,
-      ],
-      null,
-    ]
-  }
->
-  <View
-    accessibilityComponentType="button"
-    accessibilityLabel="Lorem ipsum"
-    accessibilityTraits={
-      Array [
-        "button",
-      ]
-    }
-    accessible={true}
-    hasTVPreferredFocus={undefined}
-    hitSlop={undefined}
-    isTVSelectable={true}
-    nativeID={undefined}
-    onLayout={undefined}
-    onResponderGrant={[Function]}
-    onResponderMove={[Function]}
-    onResponderRelease={[Function]}
-    onResponderTerminate={[Function]}
-    onResponderTerminationRequest={[Function]}
-    onStartShouldSetResponder={[Function]}
-    style={
-      Array [
-        Object {
-          "backgroundColor": "transparent",
-        },
-        Array [
-          Array [
-            Object {
-              "borderRadius": 40,
-              "height": 32,
-              "paddingBottom": 8,
-              "paddingLeft": 12,
-              "paddingRight": 12,
-              "paddingTop": 8,
-            },
-            Object {
-              "borderColor": "transparent",
-            },
-          ],
-          Object {},
-        ],
-      ]
-    }
-    testID={undefined}
-    tvParallaxProperties={undefined}
-  >
-    <View
-      style={
-        Array [
-          Object {
-            "alignItems": "center",
-            "flex": 1,
-            "flexDirection": "row",
-            "justifyContent": "center",
-          },
-          undefined,
-        ]
-      }
-    >
-      <Text
-        accessible={true}
-        allowFontScaling={true}
-        disabled={false}
-        ellipsizeMode="tail"
-        style={
-          Array [
-            Object {
-              "color": "rgb(82, 76, 97)",
-              "fontFamily": "System",
-              "fontSize": 13,
-              "fontWeight": "400",
-              "lineHeight": 18,
-            },
-            Object {
-              "fontWeight": "600",
-            },
-            Array [
-              Array [
-                Object {
-                  "backgroundColor": "transparent",
-                  "color": "rgb(255, 255, 255)",
-                },
-                Object {
-                  "color": "rgb(255, 255, 255)",
-                },
-              ],
-              Object {},
-            ],
-          ]
-        }
-      >
-        Lorem ipsum
-      </Text>
-    </View>
-  </View>
-</BVLinearGradient>
-`;
-
```

Comment:
I haven't removed tests for buttons that have both text and an icon, because as discussed we're not removing it, just not advertising the functionality.